### PR TITLE
applications: nrf5340_audio: Add metadata for ble5-ctr-rpmsg build

### PR DESCRIPTION
**Pull request by Frodevan**
Adds a manifest.yaml file with metadata for the controller used
with the nRF5340_audio application.

Signed-off-by: Frode van der Meeren <frode.vandermeeren@nordicsemi.no>

### CODE REVIEW

**Review by alexsven — APPROVED** [review]

File: applications/nrf5340_audio/bin/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+description: Packetcraft Controller
```
Comment (rick1082) [inline applications/nrf5340_audio/bin/manifest.yaml:1]: Shall we use the name "LE Audio Controller Subsystem for nRF53" which shown in the doc?
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/1.9.99-dev1/nrf/applications/nrf5340_audio/README.html#legal-notices-and-disclaimers

Comment (erikrobstad) [reply applications/nrf5340_audio/bin/manifest.yaml:1]: To my understanding "LE Audio Controller Subsystem for nRF53" is the name we should use. This name is consistently used in the documentation.

**Review by erikrobstad — CHANGES_REQUESTED** [review]
Change description to "LE Audio Controller Subsystem for nRF53"

File: applications/nrf5340_audio/bin/manifest.yaml (hunk 1 above)
Comment (Frodevan) [reply applications/nrf5340_audio/bin/manifest.yaml]: Updated

**Review by rick1082 — APPROVED** [review]

**Review by erikrobstad — APPROVED** [review]

**Pull request merged**